packages/dds/tree/api-report/tree.api.md

```diff
@@ -1586,7 +1586,7 @@ export class SharedTree implements ITree {
     // (undocumented)
     getAttachSummary(fullTree?: boolean | undefined, trackState?: boolean | undefined, telemetryContext?: ITelemetryContext | undefined): ISummaryTreeWithStats;
     // (undocumented)
-    static getFactory(): TreeFactory;
+    static getFactory(): IChannelFactory;
```

Comment:
Avoid exposing 'TreeFactory'.  AzureClient blindly casts the IChannel to the type of the class from which it received its factory (via .getFactory()).

Note that despite it being the prevailing pattern in our code, calling 'runtime.create(..)' or 'IChannelFactory.create(..)' directly is not publicly supported by the SDK.